3.9/graphs-pregel-algorithms.md

```diff
@@ -0,0 +1,319 @@
+---
+layout: default
+description: Pregel algorithms can be used for graph exploration, path finding, analytics queries, and much more. 
+---
+# Pregel Algorithms
+
+{{ page.description }}
+{:class="lead"}
+
+Pregel algorithms are used in scenarios where you need to do an
```

Comment:
The term "Pregel" is used in our docs both in the sense of a computation model and of our implementation of this model. For me, it's not always clear what meaning is being used. Because we cannot change the well-known term "Pregel" as the computation model, I suggest to use another term at least in the documentation, e.g., "Arango-Pregel" (this should probably be discussed with our management).